### ISSUE TITLE
Reverse Controls?

### ISSUE THREAD

**Issue by AmazingSpanoMan** [kerfuffle/FTC-9533#2]
Any way to add a toggle button to the driver one controller that make all the controls reverse?
In other words. pushing a toggle button on the controller will now make the back of the robot the front and so forth.

The thought is that we can put the debris gathering device on the back of the bot and the toggle will allow use to drive the robot in reverse without having to actually reverse the controls on the gamepad.

**Comment by kerfuffle** [comment]
Yeah that's possible, just have to make sure the driver makes a mental note as to what state they are in.
